crates/host/src/wasmbus/mod.rs

```diff
@@ -48,14 +48,16 @@ use uuid::Uuid;
 use wascap::jwt;
 use wasmcloud_control_interface::{
     ActorAuctionAck, ActorAuctionRequest, ActorDescription, HostInventory, LinkDefinition,
-    ProviderAuctionRequest, ProviderDescription, RemoveLinkDefinitionRequest, ScaleActorCommand,
-    StartActorCommand, StartProviderCommand, StopActorCommand, StopHostCommand,
+    LinkDefinitionList, ProviderAuctionRequest, ProviderDescription, RemoveLinkDefinitionRequest,
+    ScaleActorCommand, StartActorCommand, StartProviderCommand, StopActorCommand, StopHostCommand,
     StopProviderCommand, UpdateActorCommand,
 };
 use wasmcloud_runtime::capability::{messaging, Bus, KeyValueReadWrite, Messaging};
 use wasmcloud_runtime::{ActorInstancePool, Runtime};
 
 const SUCCESS: &str = r#"{"accepted":true,"error":""}"#;
+const CLAIMS_PREFIX: &str = "CLAIMS_";
+const LINKDEF_PREFIX: &str = "LINKDEF_";
```

Comment:
these are only used once in the whole codebase, is there really any benefit we get from not just inlining the values?